pkg/quota/generic/evaluator.go

```diff
@@ -41,6 +41,9 @@ func ListResourceUsingInformerFunc(f informers.SharedInformerFactory, resource s
 		if err != nil {
 			return nil, err
 		}
+		if !informer.Informer().HasSynced() {
```

Comment:
the change is fine, i am just curious if this actually has happened.  i thought we had checks elsewhere in the code that ensured all informers had been synced.